packages/rpc-graphql/README.md

```diff
@@ -3,61 +3,1160 @@
+                                amount: expect.any(String),
+                                authority: {
+                                    address: expect.any(String),
+                                    lamports: expect.any(BigInt),
+                                },
+                                destination: {
+                                    data: {
+                                        parsed: {
+                                            info: {
+                                                address: expect.any(String),
+                                                mint: data: {
+                                                    parsed: {
+                                                        info: {
+                                                            address: expect.any(String),
+                                                            decimals: expect.any(Number),
+                                                        }
+                                                    }
+                                                },
+                                                owner: {
+                                                    address: expect.any(String),
+                                                    lamports: expect.any(BigInt),
+                                                },
+                                            }
+                                        }
+                                    }
+                                },
+                                source: {
+                                    data: {
+                                        parsed: {
+                                            info: {
+                                                address: expect.any(String),
+                                                mint: data: {
+                                                    parsed: {
+                                                        info: {
+                                                            address: expect.any(String),
+                                                            decimals: expect.any(Number),
+                                                        }
+                                                    }
+                                                },
+                                                owner: {
+                                                    address: expect.any(String),
+                                                    lamports: expect.any(BigInt),
+                                                },
+                                            }
+                                        }
+                                    }
+                                },
+                            },
+                            program: 'spl-token',
+                        },
+                    }])
+                },
+            },
+        },
+    },
+});
```

Comment:
This is awesome! Another one where seeing the output would be really nice IMO

Reply:
Added!